homeassistant/components/webostv/quality_scale.yaml

```diff
@@ -0,0 +1,61 @@
+rules:
+  # Bronze
+  action-setup: todo
+  appropriate-polling: todo
+  brands: done
+  common-modules:
+    status: exempt
+    comment: The integration does not use common patterns.
+  config-flow-test-coverage: todo
+  config-flow: todo
+  dependency-transparency: todo
+  docs-actions: todo
+  docs-high-level-description: todo
+  docs-installation-instructions: todo
+  docs-removal-instructions: todo
+  entity-event-setup: todo
+  entity-unique-id: todo
+  has-entity-name: todo
+  runtime-data: todo
+  test-before-configure: todo
+  test-before-setup: todo
+  unique-config-entry: todo
+
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: todo
+  docs-configuration-parameters: todo
+  docs-installation-parameters: todo
+  entity-unavailable: todo
+  integration-owner: todo
+  log-when-unavailable: todo
+  parallel-updates: todo
+  reauthentication-flow: todo
+  test-coverage: todo
```

Comment:
Your `client` just uses a normal `patch`, I don't think we have to use `monkeypatch` to set attributes here.

```py
    monkeypatch.setattr(client, "is_connected", Mock(return_value=False))
    monkeypatch.setattr(client, "connect", Mock(side_effect=WebOsTvPairError))
```
Could be
```py
client.is_connected.return_value = False
client.connect.side_effect=WebOsTvPairError
```